### DISCUSSION TITLE
learning-in-digital-world/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#48] [Announcements]
# learning-in-digital-world/

چند روز پیش داشتم با یکی از آشنایان در مورد کنکور صحبت می‌کردم. گفتم بد نیست، اینجا هم بنویسم.
انتظار من اینه که صحبت‌های من رو در حد صحبت‌هایی که تو استخر با بغل دستی انجام میشه، در نظر بگیرید و اگر ازش خوشتون نمیاد به سادگی عبور کنید. کما این که واقعاً هم چند روز پیش تو استخر همین بحث‌ها پیش اومد :)
قبل از اون لازمه به مسئله‌ای اشاره کنم که امیدوارم، آموزش دیجیتال رو با آموزشی که در دوران کرونا توسط مدارس و دانشگا‌ه‌ها انجام شد، اشتباه نگیرید.

https://aprd.ir/learning-in-digital-world/

**Comment by elhamghadiri** [comment]
مطلب خوبی بود و کاملا موافقم . با اینکه در ابتدا فکر میکردم احتمالا نکته تازه‎‌ای برای من نخواهد داشت اما زاویه دید شما به این مسئله برام تازگی داشت و یاد گرفتم. ممنون

**Reply by pourmand1376** [reply]
ممنون از توجه شما. 

راستش اون کسی که موقع نوشتن مدنظرم بود، یک آدم دم کنکوری مثل خودم بود و می‌خواستم بهش بگم که این فضا رو دستکم نگیره. اون زمان فکر نمی‌کردم یک «مادر» هم بتونه این مطلب رو بخونه و براش مفید باشه. 

پی‌نوشت: تو اینترنت سرچ کردم و [وبلاگ مفیدتون](https://eghadiri.ir/) رو دیدم و از خوندن مطالب‌تون لذت بردم.

**Reply by elhamghadiri** [reply]
ممنون از لطفتون.
راستش خیلی وقته مطلب جدیدی ننوشتم اما با خوندن مطالب دوستانم که شما هم یکی از آنها هستید تصمیم دارم که مجدد شروع به نوشتن کنم.

**Comment by mmrzax** [comment]
جالب بود برام، حداقل الان در موقعیتی هستم که باید این رو میخوندم =))، مطلبی از شما روی ویرگول دیدم و اسمتون رو سرچ کردم و به این صفحه رسیدم.

(دانشجوی برقی که امسال میخواد برای ارشد کامپیوتر بجنگه (به صورت **دیجیتال** البته)

**Comment by mustafa-sadeghi** [comment]
سلام وقت عالی بخیر
در درس مهارت یادگیری در متمم مطلب جالبی در اینباره خوندم ،با توسعه ابزارهای دیجیتال روند جهانی اینه که از آموزش متمرکز در مدرسه و دانشگاه داریم می ریم به سمت یادگیری فردی.
الان در مورد هر مبحث درسی جستجو کنی کلی محتوا از بهترین اساتید در سطح کشور و جهان در قالب های مختلف ویدیو ، بلاگ و دوره و ... در حدی که انگار شوک محتوایی که مارک شافر می گفت در آموزش هم اتفاق افتاده و دانشجو یا دانش آموز نمیدونه از کدوم دوره یا منبع استفاده کنه.
برای همین بنظرم هزینه های میلیونی که برای کلاس ها کنکور پرداخت می شه ضروری نیست و مهم یادگیری مهارت یادگیریه.
یکی از ابزارهای مهم دیگه در آموزش همین مدل های زبانی بزرگ هستند و حتی در همون حالت جنرال می تونن نقش یک مربی را ایفا کنن.شنیدم Andrej Karpathy یک مدرسه هوش مصنوعی راه انداخته که از ابتدای دوره یک مربی هوش مصنوعی کنارت هست و در فرآیند دوره کمک ات می کنه.

لذا زنده باد آموزش دیجیتال!
